papis/commands/export.py

```diff
@@ -144,13 +155,30 @@ def cli(query: str,
     ret_string = run(documents, to_format=fmt)
 
     if ret_string and not folder:
-        if out is not None:
-            logger.info("Dumping to '%s'.", out)
-            with open(out, "a+") as fd:
-                fd.write(ret_string)
-        else:
+        if out is None:
             logger.info("Dumping to STDOUT.")
             click.echo(ret_string)
+            return
+
+        mode = "a" if append else "w"
+
+        if os.path.exists(out):
+            if append:
+                msg = f"Appending to '{out}'"
```

Comment:
```suggestion
                msg = f"Appending to '{out}'."
```
I think most logger messages have a full stop at the end, so it's good to be consistent.

Reply:
Thank you, I accidentally dropped the dots while refactoring

Force-amended to the refactor commit